src/shared/components/Cart/styles.ts

```diff
@@ -0,0 +1,91 @@
+import styled from 'styled-components';
+import { Media } from 'react-bootstrap';
+
+import Colors from 'shared/colors';
+import media from 'shared/media-types';
+
+type CartWrapperProps = {
+  size: 'small' | 'medium' | 'large';
+};
+type CartSubHeaderProps = {
+  textColor?: string;
+};
+type CartShapesProps = CartWrapperProps & {
+  violet?: boolean;
+};
+
+export const CartWrapper = styled.div<CartWrapperProps>`
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+  padding-left: 20px;
+
+  div:first-child {
+    ${({ size }): string => (size === 'large' ? 'margin-right: 60px' : '')};
```

Comment:
use types for `size`

Reply:
I used the 'generics' approach: 
```javascript
styled.div<CartWrapperProps>``;
``` 
Do you think it's a bad idea?

Reply:
Yes, we should be strict here and use  defined `enums`, e.g. `size === ${Size.LARGE}`. We do not have to use 'string' values at all - as it can cause a lot of confusing and mix-ups in future.

Other benefit is it's easier to manage and if we update enum value, we can update it just in one place.